watertap/data/techno_economic/co2_addition.yaml

```diff
@@ -0,0 +1,20 @@
+default:
+  energy_electric_flow_vol_inlet:
+    value: 0.0
+    units: kWh/m^3
+  specific_capital_cost_flow_vol_inlet:
+    base_fixed_cap_cost:
+      value: 0.464
+      units: M$
+    cap_exp_par:
+      value: 0.7
+      units: dimensionless
+    vol units:
+      value: Mgallons/day
+    capital cost units:
+      value: M$
+    year:
+      value: 2019
+    cost_function_form:
+      function f(flow_vol_in): base_fixed_cap_cost*flow_vol_in^cap_exp
```

Comment:
If we do change `base_fixed_cap_cost`, we'd have to change it here too

Reply:
Yes, and maybe a bigger question for @andrewlee94 - should I be updating .yaml files with the costing parameters?

Reply:
```suggestion
      function f(flow_vol_in): cap_basis_par*flow_vol_in^cap_exp_par
```

Reply:
Not yet - a lot of this information isn't actually needed, and we need to settle on the naming conventions which would best be done when looking at the costing code.